arch/risc-v/src/common/riscv_vectors.S

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * arch/risc-v/src/fe310/fe310_vectors.S
+ * arch/risc-v/src/common/riscv_vectors.S
```

Comment:
Why we do not have `.global __reset_vec` and `__reset_vec` implementation is removed?

Reply:
I'm not sure if that matters, but just saw that some files had it

Reply:
Oh , __reset_vec is used to keep the entry at start position at the flash, let's adjust the HEAD_ASRC to remove it.

Reply:
I'm asking because I saw
```
  .section .vectors, "ax"
  .option norvc
  j exception_common  /* 0  */
  j exception_common  /* 1  */
  j exception_common  /* 2  */
  j exception_common  /* 3  */
...
```
in `rv32m1_vectors.S`. Also `KEEP(*(.vectors))` is available in `boards/risc-v/rv32m1/rv32m1-vega/scripts/ld.script`, but I do not see where `.section .text.vec` is used.

Reply:
Yes, the `.vec` seems unused and can be removed.

Reply:
Probably will need to clean-up linker scripts for corresponding boards as well in a separate PR